server/network/aoprotocol.py

```diff
@@ -549,31 +591,36 @@ def net_cmd_ms(self, args):
                 self.client.send_ooc('You don\'t any areas!')
                 return
             text = ' '.join(part[1:])
+            send_args[4] = text
         elif text.startswith('/testify '): # Start a new testimony in this area.
             part = text.split(' ')
             text = ' '.join(part[1:]) # remove command
+            send_args[4] = text
```

Comment:
why is this line meaningful if send_args[4] is modified immediately afterward again